.devcontainer/postStartCommand.sh

```diff
@@ -2,13 +2,21 @@
 #git remote add upstream https://github.com/twjudson/aata
 #git remote add origin https://github.com/jbrock1999h/aata
 
+# The codespace tends to start under the main branch by default, and if the remotes aren't set up correctly you cannot change it to master. This is the line of code that would allow you to automatically set the branch to master if you have origin set (if you know the URL for the user's repo):
+#git checkout --track origin/master
+
 # Reset to origin main (This is for if the codespace has not caught up to changes made previously)
 #git pull origin main
 #git reset --hard origin/main
 
 # Build pretext project
-#pretext build
-#pretext generate
-pretext view web &
+pretext build
```

Comment:
This belongs in the Docker build, not here, since it takes a while to complete. The same is true for the next line.